applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -432,7 +432,6 @@ validate_account(JObj, Context) ->
                                  ,[{fun cb_context:set_req_data/2, JObj}
                                   ,{fun cb_context:set_req_nouns/2, [{?KZ_ACCOUNTS_DB, []}]}
                                   ,{fun cb_context:set_req_verb/2, ?HTTP_PUT}
-                                  ,{fun cb_context:set_resp_status/2, success}
```

Comment:
(I removed your other comment cause it's exactly the same as this one)
See my comment about using record preset above.